src/cli/backup/exchange.go

```diff
@@ -30,6 +30,7 @@ import (
 var (
 	backupID     string
 	exchangeData []string
+	site         []string
```

Comment:
nit: it feels a little odd to have this in the exchange file. OneDrive seems to have it's own set of globals in it's file as well

Reply:
Ah, good call.